server/src/main/java/org/opensearch/extensions/settings/SettingsRequestHandler.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.extensions.settings;
+
+import org.opensearch.common.settings.Setting;
+import org.opensearch.common.settings.SettingsModule;
+import org.opensearch.extensions.ExtensionStringResponse;
+import org.opensearch.transport.TransportResponse;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Handles requests to register extension settings.
+ *
+ * @opensearch.internal
+ */
+public class SettingsRequestHandler {
```

Comment:
`SettingRequestHandler` here

Reply:
Similar to above. I clarified class javadoc with the words "list of custom extension settings".